create_redistributable/pkg/elcapitan-10.11_build

```diff
@@ -1 +1 @@
-37
\ No newline at end of file
+38
```

Comment:
This version change was a mistake. But a legitimate one. It was incremented by a build (a successful one I might add) on an El Capitan machine using this fork.